pkg/storage/sos/object.go

```diff
@@ -31,6 +31,28 @@ import (
 	"github.com/exoscale/cli/utils"
 )
 
+// BatchErrorList contains list of errors parsed from AWS SDK types.DeletedObject Error attribute.
+// Only Message attribute of each types.Error object is taken and converted to std Error.
+type BatchErrorList struct {
+	List []error
+}
+
+func NewBatchErrorList() *BatchErrorList {
```

Comment:
you can probably reuse the already imported package in the CLI:
https://github.com/exoscale/cli/blob/master/cmd/output.go#L8